### PR TITLE
Force v for RemoteTech.netkan

### PR DESCRIPTION
**Pull request by plague006**
Less an issue for 1.0.4, but an issue that means there are no releases for 1.0.2.

### CODE REVIEW

**Comment by plague006** [comment]
Can one of the admins verify this patch? Where's that slacker @netkan-bot asking the important questions?

File: NetKAN/RemoteTech.netkan [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	"license": "restricted",
 	"release_status": "stable",
 	"x_netkan_license_ok": true,
+    "x_netkan_force_v": 1,
```
Comment (dbent) [inline NetKAN/RemoteTech.netkan]: Value should be `true`, not `1`.

**Comment by plague006** [comment]
I didn't give it a second thought when it passed netkan and installed.

**Comment by dbent** [comment]
:+1:

**Pull request merged**